src/utils.ts

```diff
@@ -94,17 +125,18 @@ function toLogEntry(item: Record<string, any>): Record<string, any> {
   const type = item.type
   const timestamp = item.time || new Date().getTime()
 
-  const cleanedItem = _.omit(item, [
-    "time",
-    "level",
-    "msg",
-    "hostname",
-    "service",
-    "pid",
-    "stack",
-    "type",
-  ])
-  const context = _.pickBy({ host, service, pid, stack, type }, (x) => x)
+  const {
+    time: _time,
+    level: _level,
+    msg: _msg,
+    hostname: _hostname,
+    service: _service,
+    pid: _pid,
+    stack: _stack,
+    type: _type,
+    ...cleanedItem
+  } = item
```

Comment:
There's a couple of ways to remove properties from an object, this is just one method I'm used to using, and aligns fairly nicely with how `_.omit` works. Let me know if you'd rather I use something else like `delete`